pkg/kubelet/runonce.go

```diff
@@ -85,27 +85,27 @@ func (kl *Kubelet) runOnce(pods []*v1.Pod, retryDelay time.Duration) (results []
 		}(pod)
 	}
 
-	klog.Infof("Waiting for %d pods", len(admitted))
+	klog.InfoS("Waiting for pods", "numPods", len(admitted))
 	failedPods := []string{}
 	for i := 0; i < len(admitted); i++ {
 		res := <-ch
 		results = append(results, res)
 		if res.Err != nil {
-			faliedContainerName, err := kl.getFailedContainers(res.Pod)
+			failedContainerName, err := kl.getFailedContainers(res.Pod)
 			if err != nil {
-				klog.Infof("unable to get failed containers' names for pod %q, error:%v", format.Pod(res.Pod), err)
+				klog.InfoS("Unable to get failed containers' names for pod", "pod", klog.KObj(res.Pod), "err", err)
 			} else {
-				klog.Infof("unable to start pod %q because container:%v failed", format.Pod(res.Pod), faliedContainerName)
+				klog.InfoS("Unable to start pod because container failed", "pod", klog.KObj(res.Pod), "containerName", failedContainerName)
 			}
 			failedPods = append(failedPods, format.Pod(res.Pod))
 		} else {
-			klog.Infof("started pod %q", format.Pod(res.Pod))
+			klog.InfoS("Started pod", "pod", klog.KObj(res.Pod))
 		}
 	}
 	if len(failedPods) > 0 {
 		return results, fmt.Errorf("error running pods: %v", failedPods)
 	}
-	klog.Infof("%d pods started", len(pods))
+	klog.InfoS("Pods started", "podNum", len(pods))
```

Comment:
```suggestion
	klog.InfoS("Pods started", "numPods", len(pods))
```
We should be consistent in naming arguments, above you use "numPods"